CONTRIBUTING.org

```diff
@@ -20,11 +20,23 @@ make plain
 #+end_src
 
 * Contributing code
-The basic code style guide is to use:
+Here are the important settings:
 #+begin_src elisp
 (setq indent-tabs-mode nil)
+(setq lisp-indent-function #'common-lisp-indent-function)
 #+end_src
-It is provided for you in [[https://github.com/abo-abo/swiper/blob/master/.dir-locals.el][.dir-locals.el]], please obey it.
+
+The setting for =indent-tabs-mode= is automatically applied by
+=.dir-locals.el=. In Emacs 26, you can apply the setting for
```

Comment:
What is the significance of Emacs version 26? Doesn't the suggestion work in prior versions?

Nitpick: please honour the `sentence-end-double-space` setting in the project's `dir-locals-file`.

Reply:
`.dir-locals-2.el` support was added only in Emacs 26. I'll fix the sentence spacing, sorry about that.

Reply:
Wait, on second thought—the rest of the file _also_ uses single spaces after periods! I could fix the rest of it as well, though.

Reply:
> `.dir-locals-2.el` support was added only in Emacs 26

Right, but there's never been something stopping users from modifying `.dir-locals.el` directly. My point is that the current wording suggests the entire recipe (`(add-to-list ...)` included) is specific to Emacs 26, when only the `.dir-locals-2.el` suggestion is.